frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-metrics/charts/successRatioDonut.tsx

```diff
@@ -0,0 +1,39 @@
+import { ChartDonut, ChartDonutProps, ChartThemeColor } from '@patternfly/react-charts';
+import * as React from 'react';
+
+interface SuccessRatioDonutProps {
+  totalValue: number;
+  successValue: number;
+}
+
+const SuccessRatioDonut: React.FC<SuccessRatioDonutProps & ChartDonutProps> = ({
+  data,
+  width,
+  totalValue = 0,
+  successValue = 0,
+  ariaDesc,
+  ariaTitle,
+}) => {
+  return (
+    <ChartDonut
+      ariaDesc={ariaDesc}
+      ariaTitle={ariaTitle}
+      constrainToVisibleArea
+      data={data}
+      sortKey={successValue ? ['success', 'failed'] : ['failed']}
+      labels={({ datum }) => `${datum.x}: ${datum.y}%`}
+      colorScale={successValue ? [ChartThemeColor.green, '#c9190b'] : ['#c9190b']}
+      padding={{
+        bottom: 20,
+        left: 20,
+        right: 20,
+        top: 20,
+      }}
+      subTitle={data ? `${successValue} succeeded of ${totalValue}` : ''}
```

Comment:
I18n, follow the example used in [PipelineBuildDecoratorTooltip.tsx](https://github.com/openshift/console/blob/master/frontend/packages/pipelines-plugin/src/topology/build-decorators/PipelineBuildDecoratorTooltip.tsx#L44) (for instance)